server/repository/src/database_settings.rs

```diff
@@ -46,6 +56,17 @@ impl DatabaseSettings {
     pub fn connection_string_without_db(&self) -> String {
         self.connection_string()
     }
+
+    pub fn full_init_sql(&self) -> Option<String> {
+        //For SQLite we want to enable the Write Head Log on server startup
+        if let Some(sql_statement) = &self.init_sql {
```

Comment:
match self.init_sql.map {
    None => Some(SQLITE_WAL_PRAGMA.to_string()),
    Some => ... (as per your logic, although do we need end_width check, would `;;` be legit ?)
}

Reply:
this is :nit btw

Reply:
also, we've been using options methods like `map`, and `or`, which would work here, but again stylistic

Reply:
Looks like `;;` works fine. I'll go with that and add additional logic if needed. Was trying to be defensive but if you specify a broken init_sql you'll have issues on startup anyway so I think it's probably cleaner to just send a double `;;` it for now.

Reply:
I like the match in this case.

I general I'm a fan of early returns so will probably tend to favour them sometimes even when it's not the cleanest option. Always happy for feedback and want to learn style preferences.

Reply:
Early returns are great, i do like them, like if you had to use the value of some in the rest of the method but return None otherwise